scarlet/observation.py

```diff
@@ -434,6 +414,7 @@ def match(self, model_frame):
             model_frame.wcs,
             self.frame.wcs,
             isrot=self.isrot,
+            perimeter = perimeter
```

Comment:
I don't think this is the way to do it. `perimeter` (or better: `overlap`) needs to affect the construction of the model frame, not the matching of the model frame and the observed frame. The latter really only needs the  intersection plus the width of the diff kernel. This will have to be dealt with in another PR, which also ties in #120. See my code recommendation there.